docs/Introduction/getting-started.md

```diff
@@ -0,0 +1,169 @@
+          <a href="/docs/intermediates-tutorial/">
+            <div class="arrowed-text">
+                Learn More
+                <img class="cta-learnmore-arrow" src="/images/misc/navigation-arrow-right.svg" loading="lazy" alt="Right arrow">
+            </div>
+          </a>
+        </div>
+      </div>
+    </div>
+    <div class="cl-featuredcard">
+      <div>
+        <div class="card-icon-wrapper">
+          <img src="/images/misc/5f75b5e6e24b376ed9ab5c1e_programming-browser.svg" class="cl-image-featured"></img>
+        </div>
+        <h2>Already familiar with smart contract basics?</h2>
+        <p>
+          Dive deeper into Chainlink product and services on Ethereum chains.
+        </p>
+        <a href="/ethereum/" class="cl-button--ghost">
+          Chainlink on Ethereum Chains
+        </a>
+      </div>
+    </div>
+    <div class="cl-section">
+      <div class="cl-section cl-section--tools">
+        <h3>Explore Additional Resources</h3>
+        <p>Find all the right resources to get the most out of Chainlink.</p>
+        <div style="display: flex;">
+          <a class="cl-productcard" href="/docs/architecture-overview/" target="_blank">
+            <div>
+                <img
+                  src="/images/misc/5f85d14b6782933bd5a70060_decentralized.svg"
+                  width="150px"
+                />
+                <h3>Architecture Overview</h3>
+            </div>
+          </a>
+          <a class="cl-productcard" href="/docs/other-tutorials/" target="_blank">
+            <div>
+                <img
+                  src="/images/misc/5f75e7d3f8ef05758ce8c1f5_bulb.svg"
+                  width="150px"
+                />
+                <h3>Videos and Tutorials</h3>
+            </div>
+          </a>
+          <a class="cl-productcard" href="https://blog.chain.link/" target="_blank">
+            <div>
+                <img
+                  src="/images/misc/61163fe987f45b67ab88bcfe_common-file-text-chainlink.svg"
+                  width="150px"
+                />
+                <h3>Chainlink Blog</h3>
+            </div>
+          </a>
+          <a class="cl-productcard" href="/docs/getting-help/" target="_blank">
+            <div>
+                <img
+                  src="/images/misc/help_avatar.svg"
+                  width="150px"
+                />
+                <h3>Get Support</h3>
+            </div>
+          </a>
+          <a class="cl-productcard" href="https://chain.link/community" target="_blank">
+            <div>
+                <img
+                  src="/images/misc/5f75ccd99b524261854d093d_multiple-users-network.svg"
+                  width="150px"
+                />
+                <h3>Join the Community</h3>
+            </div>
+          </a>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+
+Running your own Chainlink node? See the [Node Operator's Guide](/chainlink-nodes/) to learn about starting and maintaining Chainlink Nodes.
```

Comment:
This looks like it only applies to folks who are already running a node, what about people who want to run nodes?

Reply:
Restored the original text. "Need to run your own Chainlink Node?"